src/components/screens/Lightning/payInvoice.tsx

```diff
@@ -380,7 +380,7 @@ export default function PayInvoicePage({ navigation, route }: TPayLNInvoicePageP
 				{!input.length && !isKeyboardOpen &&
 					<TouchableOpacity style={{ marginVertical: 10 }} onPress={() => {
 						void (async () => {
-							await openUrl('lightning://')
+							await openUrl('lightning://')?.catch((err: string) => openPromptAutoClose({ msg: err }) )
```

Comment:
Same

Reply:
Have you checked msg data type in your interface? It is string. Right?